event/src/unvalidated/signed/mod.rs

```diff
@@ -192,6 +197,34 @@ impl Envelope {
                 })
         })
     }
+
+    /// Get the signature
+    pub fn get_signature(&self) -> &Bytes {
+        &self.signatures[0].signature
+    }
+
+    /// Get the signed payload
+    pub fn get_payload(&self) -> &Bytes {
+        &self.payload
+    }
+
+    /// Construct the jws header from the signature protected bytes
+    pub fn get_header(&self) -> Result<ssi::jws::Header, anyhow::Error> {
```

Comment:
we don't usually include `get` in the names.. maybe call this `jws_header()` or `header_as_jws`? same about removing `get` from `get_payload` and `get_signature`